gradle.properties

```
@@ -54,7 +54,7 @@ stdlibWebsubhubVersion=0.2.0-alpha9-20210503-103800-141913c
 stdlibSqlVersion=0.6.0-alpha10-20210503-112100-4c91ca0
 
 # Dev Tools JS Libraries
-devToolsVersion=0.10.3-SNAPSHOT
+devToolsVersion=0.10.3
```

```suggestion
devToolsVersion=0.10.4
```
Shouldnt it be 0.10.4?